### ISSUE TITLE
authorization example

### ISSUE THREAD

**Issue by scottiejibbs** [eddiemoore/angular-spotify#46]
Eddie - thank you so much for this angular service.  As a newb to web development this has been huge in motivating me to learn more.  I've hit a bit of a snag on the authorization example - I'm not certain if my mistake or not.
Login opens the dialog window, I login to spotify with my credentials. 
When the example's callback.html is triggered I see that the spotify-token is correctly parsed and set on the localStorage.
But I never see the added event listener trigger the storageChanged() method.
Therefore the dialog window hangs open, and auth is never considered complete.
I've tried firefox and chrome and see the same issue.
Any idea what might be going on? 
Any help is much appreciated.
Thanks
-Scott

**Comment by eddiemoore** [comment]
Hi @scottiejibbs Thanks for trying out angular-spotify. I will investigate and see if I can find the problem.

**Comment by scottiejibbs** [comment]
My problem solved @eddiemoore!  And of course, it was self inflicted.  I had mistakenly launched the example index.html as a file:///, instead of directing my browser to "http://localhost/index.html" :-(  When clicking login button, the dialog window opened to the appropriate "http://localhost/callback.html#....."  The localStorage was updated, but the localStorage changed event will only be seen by *other* windows on the *same* domain, as I discovered after some reading and testing.  Doooh!!!  By properly directing the browser to "http://localhost/index.html", repeating the login, it all worked as expected.  Hope you didn't burn much time investigating.  Thanks for the quick response.
